tls/test_tls_handshake.py

```diff
@@ -239,22 +267,115 @@ def ADDED_CHANGECIPHERSPEC(self):
             @ATMT.condition(ADDED_CHANGECIPHERSPEC)
             def should_add_ClientFinished(self):
                 self.add_record()
-                self.add_msg(TLSApplicationData(data=f"GET / HTTP/1.1\r\nHost: tempesta-tech.com\r\n\r\n"))
+                self.add_msg(
+                    TLSApplicationData(data=f"GET / HTTP/1.1\r\nHost: tempesta-tech.com\r\n\r\n")
+                )
                 self.add_record()
                 self.add_msg(TLSFinished())
                 raise self.ADDED_CLIENTFINISHED()
 
         conn = TlsHandshake()
         res = conn.do_12(automaton=_ModifiedTLSClientAutomaton)
         self.assertFalse(res, "Bad handshake successfully processed")
-        
-        
+
     def test_old_handshakes(self):
         self.start_all()
         res = TlsHandshakeStandard().do_old()
         self.assertTrue(res, "Wrong old handshake result: %s" % res)
 
 
+class TlsHandshakeDheRsaTest(tester.TempestaTest):
+    clients = [
+        {
+            "id": "tls-perf-DHE-RSA-AES128-GCM-SHA256",
+            "type": "external",
+            "binary": "tls-perf",
+            "cmd_args": (
+                "-c DHE-RSA-AES128-GCM-SHA256 -C prime256v1 -l 1000 -t 2 -T %s ${server_ip} 443"
```

Comment:
Please use THREADS, CONCURRENT_CONNECTIONS and DURATION from run_config.py for these params.